packages/bot/src/constants/index.js

```diff
@@ -188,6 +188,7 @@ const config = {
         [translate('1 day'), '86400'],
     ],
     mainBlocks                             : ['trade_definition', 'before_purchase', 'during_purchase', 'after_purchase'],
+    mandatoryMainBlocks                    : ['trade_definition', 'before_purchase', 'after_purchase'],
```

Comment:
This is incorrect, only `trade_definition` and `before_purchase` are mandatory blocks.